testing/trino-benchto-benchmarks/src/test/resources/statistics/tpcds/sf1000/README.md

```diff
@@ -1,2 +1,6 @@
 These statistics are copied from the hive connector from TPCDS schema with sf1000.
 So, there are no min nor max values for char-based columns.
+
+### How statistics is stored?
+- Files with `<table_name>.json` name were produced using un-partitioned tables.
```

Comment:
How were these files produced ?
We should have a standalone program here (or provide reference to some existing class if you used that) which given some existing hive catalog dumps the stats to file in json format.